src/py42/util.py

```diff
@@ -72,3 +72,30 @@ def format_dict(dict_, label=None):
     if label:
         return u"{} {}".format(label, indented_dict)
     return indented_dict
+
+
+def bool_to_str(value):
+    if isinstance(value, bool):
+        return "true" if value else "false"
+    else:
+        raise KeyError("Value must be True or False")
```

Comment:
Is `KeyError` the right error here? I'm not used to thinking of function parameters as keys.

Reply:
How about `TypeError` ?

Reply:
yeah `TypeError` is better.